Gemfile

```diff
@@ -41,6 +41,8 @@ group :development, :test do
   gem 'selenium-webdriver'
 
   gem 'rubocop'
+  gem 'danger'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem danger should appear before rubocop.